indexCap/solution.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+function capitalize(s,arr){
+  let strArr = s.split('');
+  arr.forEach( x => {if (!(x > strArr.length)) 

```

Comment:
'If' statements and their condition should be separated onto their own line by convention.
I know in CodeWars people are trying to condense things which is fine, but in production code, you wouldn't want to do a forEach and an if-condition all on the same line.